src/components/Calculator.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './Calculator.css';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <div className="row screen ">0</div>
+        <div className="row ">
+          <p className="item">AC</p>
+          <p className="item">+/-</p>
+          <p className="item">%</p>
+          <p className="item last-col">÷</p>
+        </div>
+        <div className="row ">
+          <p className="item">7</p>
+          <p className="item">8</p>
+          <p className="item">9</p>
+          <p className="item last-col">X</p>
+        </div>
+
+        <div className="row ">
+          <p className="item">4</p>
+          <p className="item">5</p>
+          <p className="item">9</p>
+          <p className="item last-col">-</p>
+        </div>
+        <div className="row ">
+          <p className="item">1</p>
+          <p className="item">2</p>
+          <p className="item">3</p>
+          <p className="item last-col">+</p>
+        </div>
+        <div className="row last row">
+          <p className="item zero">0</p>
+          <p className="item">&middot;</p>
+          <p className="item last-col">&#61;</p>
+        </div>
+      </div>
```

Comment:
- [OPTIONAL] Your design looks a bit small. It will be great to expand it a bit so that users can have view the buttons clearer and be able to interact with them.

Your design
![a1](https://user-images.githubusercontent.com/39523312/155823421-579349c0-5e61-4381-b87a-da81993d52dd.png)

Reply:
@mmsesay thank you. I will implement it.